trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    if @nameable.correct_name.length <= 10
+      @nameable.correct_name
+    else
+      @nameable.correct_name.delete(' ')
+    end
+  end
```

Comment:
- [ ] The method `correct_name` should make sure that the output of the `@nameable.correct_name` has a maximum of 10 characters. If it's longer it should trim the word. This method is not working as expected, please make sure to fix this issue. (The `Maximilianus` should be trimmed to 10 characters in the last output).

![image](https://user-images.githubusercontent.com/83803414/180045382-e8763272-645f-46ef-b4ba-a83eb434ed28.png)

Reply:
This issue is well resolved! ✔️